config/database.yml

```diff
@@ -2,6 +2,7 @@ default: &default
   adapter: postgresql
   encoding: unicode
   username: payroll
+  password: password
```

Comment:
That's how we had it originally, but we kept running into postgres issues within Github Actions if we didn't have a password - I think it doesn't let you get away with not having one, presumably for security reasons or something similar. (Because the workaround is so much more secure...)